src/clients/InsightsQueryClient.ts

```diff
@@ -621,6 +621,12 @@ class InsightsQueryClient {
     const response = await request.get();
     return response.data as Insights.InsightsQueryResponse<Insights.QueryGetAlliancesRewardsResponse>;
   }
+
+  async CrosschainVolumes(req: Insights.QueryCrosschainVolumeRequest = {}): Promise<Insights.InsightsQueryResponse<CrossChainVolume[]>> {
+    const request = this.apiManager.path("crosschain/volume", {}, req)
+    const response = await request.get()
+    return response.data as Insights.InsightsQueryResponse<CrossChainVolume[]>
+  }
```

Comment:
Think it would be good to add this query to the example script [query_insights.ts](https://github.com/Switcheo/carbon-js-sdk/blob/staging/examples/query_insights.ts)